user-guide/Advanced_Modules/DOM/DomHelper_class.md

```diff
@@ -54,6 +57,24 @@ if (!traceData.HasSucceeded())
 }
 ```
 
+### Multiple instances
+
+When multiple `DomInstances` need to get create, updated or deleted, it is recommended to call the *CreateOrUpdate* or *Delete* methods on a `DomInstance` CRUD helper component, with a list of those `DomInstances`. This feature is available from DataMiner 10.4.2/10.5.0 onwards. More information and best practices on how to use these calls is available [in these examples](xref:DOM_BulkProcessing_Example).
+
+When the operation fails for one of the `DomInstances`, the result of those calls will contain the necessary information.
+When calling `CreateOrUpdate`, the result will contain a list of `DomInstances` that were successfully created or updated. The trace data is available per token ID.
+When calling `Delete` with a list of `DomInstances`, the result will contain a list of `DomInstance` IDs that were successfully deleted. Also here the trace data is available per token ID.
+
+If an issue occurs when any item gets created or updated or deleted (e.g. validation), no exception will be thrown (even when `ThrowExceptionsOnErrorData` is true). The result of the call can used to check for which `DomInstances` the call failed. Next to that the trace data of that call is available and will contain the trace data for all processed `DomInstances`.
+
+If however the entire operation would fail (e.g. when the storage layer would fail while storing the data) a `CrudFailedException` is thrown. If, in that case, `ThrowExceptionsOnErrorData` was set to false a `DataMinerException` will be thrown instead and the `TraceData` of the last call should be checked to get more details about the issue.
+
+#### Maximum amount
+
+Since these calls might trigger related actions (such as launching script actions), it might cause a high load on the system when a lot of instances are involved. A soft limit of 100 `DomInstances` is set, to make sure those bulk operations are implemented with scalability in mind. When a higher amount of instances should be processed, consider splitting those up in batches.
```

Comment:
```suggestion
Since these calls might trigger related actions (such as launching script actions), it might cause a high load on the system when a lot of instances are involved. A limit of 100 `DomInstances` is set, to make sure those bulk operations are implemented with scalability in mind. When a higher amount of instances should be processed, consider splitting those up in batches.
```